package.json

```diff
@@ -53,12 +55,13 @@
     "@typescript-eslint/eslint-plugin": "^2.1.0",
     "@typescript-eslint/parser": "^2.1.0",
     "floss": "^3.0.0",
+    "fs-extra": "^9.0.0",
     "chai": "~3.5.0",
     "sinon": "^7.3.2",
     "sinon-chai": "^3.3.0",
     "nyc": "^14.1.1",
     "glob": "^7.1.3",
-    "jsdoc": "^3.5.5",
+    "jsdoc": "^3.6.3",
```

Comment:
Just FYI, we have intentionally not upgraded to 3.6.x of JSDoc because it breaks our docs. We are pinned to 3.5.5 for now. This is a much bigger task to upgrade. We know it doesn't work in Node 12. Eventually we'll transition to typedoc instead.